pom.xml

```
@@ -122,6 +121,12 @@
           <redirectTestOutputToFile>true</redirectTestOutputToFile>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>4.2.0</version>
+        <configuration><excludeFilterFile>${project.basedir}/spotbugs.xml</excludeFilterFile></configuration>
```

Would it make sense to rename the file?
```suggestion
        <configuration><excludeFilterFile>${project.basedir}/spotbugs-excludes.xml</excludeFilterFile></configuration>
```
I was reading the diff from bottom to top and was confused why the filter rules listed seemed to be only for rules that don't make sense (and the comments indicated I'm correct), before I saw `excludeFilterFile` here. I have little experience with FB config though, maybe it's clear to everyone else.